.circleci/config.yml

```diff
@@ -12,7 +12,8 @@ test: &test
     - run: make build-python
     - run: source .circleci/install_tvm.sh
     - run: pip install --user pytest 
-    - run: pip install --user future 
+    - run: pip install --user future
+    - run: pip install --user imageio
```

Comment:
Can you check if you are using tabs or spaces? The CI cannot run.

Reply:
I believe I have used spaces there. Re-typing them with spaces result in no `git diff` output.

Reply:
Can you try to push again or maybe revert the changes of this file? Not sure why the CI is not working.

Reply:
No it seems that line causes the problem.

Reply:
Wierd. CI is able to work now...

Reply:
> Weird. CI is able to work now...

Yeah not so sure why that glitch happened.